### PR TITLE
feat/Hot keys (#77)

### LINKED ISSUE #77: Left right hot key
Pressing the left or right hot key will move the animation forward or back. 
If animation is paused, it moves the (time and therefore map plot) forward and back in time.

### PR DESCRIPTION
**Pull request by dlinah**
# Pull Request

Closes #77

## How Has This Been Tested?

- tested manually

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: apps/nowcasting-app/components/play-button/index.tsx [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
   endTime: string;
   startTime: string;
 };
-const add30Minutes = (date: string) => {
-  var d = new Date(date);
-  d.setMinutes(d.getMinutes() + 30);
-  return d.toISOString();
-};
+
 const PlayButton: React.FC<PlayButtonProps> = ({ endTime, startTime }) => {
```
Comment (Alaatohamy) [inline apps/nowcasting-app/components/play-button/index.tsx:12]: can you auto-focus on the play button to be able to play it on click enter key (now it needs to move between page links to reach it)

File: apps/nowcasting-app/components/utils.ts [hunk 2]
```diff
@@ -51,3 +51,8 @@ export const formatISODateStringHuman = (date: string) => {
   // further formatting could be done to make it yyyy/mm/dd HH:MM
   return `${date_london} ${date_london_time}`;
 };
+export const addMinutesToISODate = (date: string, munites: number) => {
+  var d = new Date(date);
+  d.setMinutes(d.getMinutes() + munites);
+  return d.toISOString();
```
Comment (Alaatohamy) [inline apps/nowcasting-app/components/utils.ts:57]: Timezone here is zero UTC not like London timezone

File: apps/nowcasting-app/components/play-button/index.tsx (hunk 1 above)
Comment (dlinah) [reply apps/nowcasting-app/components/play-button/index.tsx:12]: great suggestion , but i don't think it belong to this story, feel free to add to the tasks

File: apps/nowcasting-app/components/utils.ts (hunk 2 above)
Comment (dlinah) [reply apps/nowcasting-app/components/utils.ts:57]: that's ok because we use utc for all data , and only convert it for displayed time and dates

**Review by Alaatohamy — APPROVED** [review]

**Pull request merged**